### PR TITLE
Add 'title' attribute to iframe for accessibility

### PR DESCRIPTION
**Pull request by Darshan-upadhyay1110**
* Resolves: # 
* Target version: main

### TODO

- [ ] ...

### Checklist

- [ ] Code is properly formatted
- [ ] Sign-off message is added to all commits
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

**Comment by Darshan-upadhyay1110** [comment]
Issue:
 The absence of a 'title' attribute for an iframe means
that users of screen readers and other assistive technologies will not be able to understand the
content and purpose of the iframe

File: src/document.js [hunk 1]
```diff
@@ -171,7 +171,7 @@ const documentsMain = {
 				+ '</form>'
 
 			// iframe that contains the Collabora Online Viewer
-			const frame = '<iframe data-cy="coolframe" id="loleafletframe" name="loleafletframe_viewer" allowfullscreen allow="clipboard-read *; clipboard-write *" nonce="' + btoa(getRequestToken()) + '" style="width:100%;height:100%;position:absolute;"/>'
+			const frame = '<iframe data-cy="coolframe" id="loleafletframe" name="loleafletframe_viewer" allowfullscreen allow="clipboard-read *; clipboard-write *" nonce="' + btoa(getRequestToken()) + '" style="width:100%;height:100%;position:absolute;" title="Collabora Online Viewer"/>'
```
Comment (eszkadev) [inline src/document.js:174]: it should be some "PRODUCT NAME" variable string I think

File: src/document.js [hunk 2]
```diff
@@ -231,7 +231,7 @@ const documentsMain = {
 				+ '</form>'
 
 			// iframe that contains the Collabora Online
-			const frame = '<iframe data-cy="coolframe" id="loleafletframe" name="loleafletframe" nonce="' + btoa(getRequestToken()) + '" scrolling="no" allowfullscreen allow="clipboard-read *; clipboard-write *" style="width:100%;height:100%;position:absolute;" />'
+			const frame = '<iframe data-cy="coolframe" id="loleafletframe" name="loleafletframe" nonce="' + btoa(getRequestToken()) + '" scrolling="no" allowfullscreen allow="clipboard-read *; clipboard-write *" style="width:100%;height:100%;position:absolute;" title="Collabora Online"/>'
```
Comment (eszkadev) [inline src/document.js:234]: same here

File: src/document.js (hunk 1 above)
Comment (eszkadev) [reply src/document.js:174]: @juliushaertl could you help here?

Comment (juliushaertl) [reply src/document.js:174]: https://github.com/nextcloud/richdocuments/blob/612379f2df0efc0002e8955538367c4b1eed0a02/src/view/Office.vue#L203 should work here as well

**Comment by juliushaertl** [comment]
Also please add this to the iframes in src/view/Office.vue

File: src/document.js (hunk 1 above)
Comment (Darshan-upadhyay1110) [reply src/document.js:174]: @juliushaertl i am not quite sure how things work in .vue files and i am confused how to load product name in [src/document.js]. can you help me here ?

Comment (Darshan-upadhyay1110) [reply src/document.js:174]: is it like 
`<iframe data-cy="coolframe" id="loleafletframe" name="loleafletframe" nonce="' + btoa(getRequestToken()) + '" scrolling="no" allowfullscreen allow="clipboard-read *; clipboard-write *" style="width:100%;height:100%;position:absolute;" title="{productName}"/>`

Comment (Darshan-upadhyay1110) [reply src/document.js:174]: oh i see i think i can import the
 `import { getCapabilities } from './services/capabilities.ts' `
 
 and use `getCapabilities().productName` as an iframe title

**Comment by juliushaertl** [comment]
After discussing with @pedropintosilva let's change the title to "Nextcloud Office - Editor for filename.odt"

**Review by eszkadev — COMMENTED** [review]
code looks ok now for me

**Review by juliushaertl — APPROVED** [review]
Pushed a small fix for the vue component as there it was not working.

getCapabilities() is no property of the component so it cannot be just used in the template.

**Comment by Darshan-upadhyay1110** [comment]
> Pushed a small fix for the vue component as there it was not working.
> 
> getCapabilities() is no property of the component so it cannot be just used in the template.

Thanks @juliushaertl 🙇🏼

**Pull request merged**